test/jax/test_sort.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+import numpy as np
+import jax
+
+from netket.jax import sort, searchsorted
+
+
+@pytest.mark.parametrize("shape", [(11, 5), (12,)])
+def test_sort(shape):
+    x = jax.random.randint(jax.random.PRNGKey(123), shape, 0, 10)
+    x_sorted = sort(x)
+    for i in range(len(x_sorted) - 1):
+        assert x_sorted[i].tolist() <= x_sorted[i + 1].tolist()
+
+
+@pytest.mark.parametrize("shape", [(11, 5), (12,)])
+def test_searchsorted(shape):
+    x = jax.random.randint(jax.random.PRNGKey(123), shape, 0, 10)
+    x_sorted = sort(x)
+
+    for i in range(len(x)):
+        k = searchsorted(x_sorted, x[i])
+        np.testing.assert_array_equal(x_sorted[k], x[i])
+
+    for i, k in enumerate(searchsorted(x_sorted, x)):
+        np.testing.assert_array_equal(x_sorted[k], x[i])
```

Comment:
can you add a test for the explicit dtype argument, and one testing that it uses np.int32 by default (I think_)